ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -390,179 +452,198 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
-                       invalid={this.state.usernameConflictError || this.usernameInvalidError()}
-                       style={{...styles.section, marginRight: '0.5rem'}}/>
-            <div style={inputStyles.iconArea}>
-              <ValidationIcon validSuccess={this.usernameValid}/>
-            </div>
-            <TooltipTrigger content={<div>Usernames can contain only letters (a-z),
-              numbers (0-9), dashes (-), underscores (_), apostrophes ('), and periods (.)
-              (minimum of 3 characters and maximum of 64 characters).<br/>Usernames cannot
-              begin or end with a period (.) and may not contain more than one period (.) in a row.
-            </div>}>
-              <InfoIcon style={{'height': '22px', 'paddingLeft': '2px'}}/>
-            </TooltipTrigger>
-            {this.state.usernameConflictError &&
-            <div style={{height: '1.5rem'}}>
-              <Error id='usernameConflictError'>
-                Username is already taken.
-              </Error></div>}
-            {this.usernameInvalidError() &&
-            <div style={{height: '1.5rem'}}><Error id='usernameError'>
-              Username is not a valid username.
-            </Error></div>}
+      <div style={{fontSize: 28, fontWeight: 400, color: colors.primary}}>Create your account</div>
+      {environment.enableAccountPages && <FlexRow>
+        <FlexColumn style={{marginTop: '0.5rem'}}>
+          <div style={{...styles.text, fontSize: 16, marginTop: '1rem'}}>
+            Please complete Step 1 of 2
           </div>
-        </Section>
-        <Section header='About you'>
-          <FlexColumn>
-            <div style={{paddingBottom: '1rem'}}>
-              <TextInput id='givenName' name='givenName' autoFocus
-                         placeholder='First Name'
-                         value={givenName}
-                         invalid={givenName.length > nameLength}
-                         style={{...styles.section, marginRight: '2rem'}}
-                         onChange={value => this.updateProfileObject('givenName', value)}/>
-              {givenName.length > nameLength &&
-              <ErrorMessage id='givenNameError'>
-                First Name must be {nameLength} characters or less.
-              </ErrorMessage>}
-              <TextInput id='familyName' name='familyName' placeholder='Last Name'
-                         value={familyName}
-                         invalid={familyName.length > nameLength}
-                         style={styles.section}
-                         onChange={v => this.updateProfileObject('familyName', v)}/>
-              {familyName.length > nameLength &&
-              <ErrorMessage id='familyNameError'>
-                Last Name must be {nameLength} character or less.
-              </ErrorMessage>}
+          <div style={{...styles.text, fontSize: 12, marginTop: '0.7rem'}}>All fields required unless indicated as optional</div>
```

Comment:
I haven't done validation on these pages yet (I have a follow up subtask on that) but I think we are trying to set the default as required on this page, because fields are far more likely to be required. Lou might have a better sense on design implications though, I can ask.

Reply:
From Lou:
```
Yes - I originally had * and response was “I don’t see them” So I made them red and response was “All I see are the *” So we decided that we state “All fields required unless indicated as optional”
Other than the Optional Demo Survey, only field optional is the professional URL I believe
```

Reply:
¯\\\_(ツ)_/¯